indico/modules/events/editing/client/js/editing/timeline/CommentForm.jsx

```diff
@@ -27,14 +33,14 @@ export default function CommentForm({onSubmit, onToggleExpand, initialValues, ex
   const onCommentClickHandler = () => {
     if (!commentFormVisible) {
       setCommentFormVisible(true);
-      onToggleExpand(true);
+      onToggleExpand && onToggleExpand(true);
```

Comment:
If you wanna be fancy :smile: 
```suggestion
      onToggleExpand?.(true);
```
or maybe even better is to set the default prop to `() => {}`. That way you can avoid all the checks